rts/at/util/src/main/java/org/jboss/jbossts/star/util/media/txstatusext/TransactionManagerElement.java

```diff
@@ -33,7 +33,7 @@
 
     @XmlElement
     public Date getCreated() {
-        return created;
+        return new Date(created.getTime());
```

Comment:
The Date object is in the schema that defines the types so its part of the specification so I cannot and do not want to change it. I could create a type that extends Date and makes it readonly but that seems excessive.